fireflower/core.py

```diff
@@ -38,6 +39,10 @@ def register_sentry(cls, sentry):
         cls.sentry = sentry
 
     @classmethod
+    def register_datadog_dogstatsd(cls, dogstatsd):
+        cls.dogstatsd = dogstatsd
+
```

Comment:
Go ahead and add the corresponding method for threadstats, so the next person to come along and want to use this isn't discouraged from using threadstats over dogstatsd:
```py
def register_datadog_dogstatsd(cls, threadstats: 'datadog.ThreadStats'):
    ...
```